src/components/DeleteFlowButton.vue

```diff
@@ -34,33 +33,14 @@ export default {
   },
   computed: {
     ...mapGetters('tenant', ['role']),
-    all() {
-      if (this.allFlows) return true
-      if (this.type === 'group') return true
-      return false
-    },
     mutationString() {
-      if (!this.all) {
-        return gql`
-          mutation($flowId: UUID!) {
-            delete_flow(input: { flow_id: $flowId }) {
-              success
-            }
-          }
-        `
-      } else {
-        let string = this.flowGroup.flows.map((flow, ind) => {
-          return `delete_${ind}: delete_flow(input: { flow_id: "${flow.id}" }) {
-              success
-            }`
-        })
-        const newString = gql`
-          mutation {
-            ${string}
+      return gql`
+        mutation($flowGroupId: UUID!) {
+          delete_flow_group(input: { flow_group_id: $flowGroupId }) {
+            success
```

Comment:
Since we don't need to dynamically build this mutation anymore, we can use a `require` call with a static mutation file

Reply:
since we decided we do want the ability to delete versions, I kept it as is (and added the other mutation in a conditional)